Resources/Locale/en-US/station-events/events/bluespace-artifact.ftl

```diff
@@ -0,0 +1 @@
+station-event-bluespace-artifact-announcement = Attention crew, our readings have determined an artifact of alien origin will shortly be appearing on the station. Please alert the research team if it is seen.
```

Comment:
you could add some flavor with regards to how/why the artifact is appearing (kind of like how the sentience event has varying flavor text)

Reply:
@EmoGarbage404 not quite what they ment, go look at the sentience event announcement